Tests/ResilientDecodingTests/ResilientTests.swift

```diff
@@ -0,0 +1,44 @@
+import ResilientDecoding
+import XCTest
+
+final class ResilientTests: XCTestCase {
+  struct Wrapper: Decodable, Hashable {
+    @Resilient var value: ResilientEnumWithFallback
+  }
+
+  func testResilientEquatable() throws {
```

Comment:
if we end up wanting to adopt this functionality, we'll want to add a separate test method per `case` in the `==` statement. These test method names should be of the form `test_equatable_whatAboutEquatableIsBeingTested()`, so this method would become `test_equatable_returnsTrueWhenValuesAreEqual()` (at least, I think that both json1 and json2 end up using `decodingFallback` so the values end up being equal? It's not the most clear what code path this unit test is testing as-is).